lib/pbench/server/database/__init__.py

```diff
@@ -1,16 +1,18 @@
-from pbench.server.database.models import *  # noqa
+from pbench.server.database.models import *  # noqa F401
```

Comment:
I'm pretty sure this one isn't under F401. In any case, I think just `import pbench.server.database.models` should work without any diagnostics to awkwardly suppress. There's just the one module namespace to import from `__init__.py` including all the model imports.

Again, another alternative would be just pulling the individual module import list up to here and avoiding the indirection.

Reply:
Yes, I addressed that in the rebase, this is a stale code